src/helpers/sorobanXdrUtils.ts

```diff
@@ -0,0 +1,79 @@
+import { StrKey, scValToNative, xdr } from "stellar-sdk";
+
+export const scValByType = (scVal: xdr.ScVal) => {
+  switch (scVal.switch()) {
+    case xdr.ScValType.scvAddress(): {
+      const address = scVal.address();
+      const addressType = address.switch();
+      if (addressType.name === "scAddressTypeAccount") {
+        return StrKey.encodeEd25519PublicKey(address.accountId().ed25519());
+      }
+      return StrKey.encodeContract(address.contractId());
+    }
```

Comment:
this is [already done](https://github.com/stellar/js-stellar-base/blob/master/src/scval.js#L309-L310) by `scValToNative`!

Reply:
Nice! This was just a quick fix to make Lab v1 functional. We'll do better for v2.